base/linalg/symmetric.jl

```diff
@@ -57,10 +57,53 @@ ctranspose(A::Hermitian) = A
 trace(A::Hermitian) = real(trace(A.data))
 
 #tril/triu
-tril(A::Hermitian,k::Integer=0) = tril(A.data,k)
-triu(A::Hermitian,k::Integer=0) = triu(A.data,k)
-tril(A::Symmetric,k::Integer=0) = tril(A.data,k)
-triu(A::Symmetric,k::Integer=0) = triu(A.data,k)
+function tril(A::Hermitian, k::Integer=0)
```

Comment:
this could be `Union{Hermitian,Symmetric}` right? The implementations look like exact copies.